TYPO3.Neos/Resources/Private/Templates/Module/Management/Workspaces/Edit.html

```diff
@@ -7,59 +7,61 @@
 
 <f:section name="content">
 	<f:form action="update" name="workspace" object="{workspace}">
-		<div class="neos-control-group">
-			<label class="neos-control-label" for="title">{neos:backend.translate(id: 'workspaces.workspace.title', source: 'Modules', package: 'TYPO3.Neos')}</label>
-			<div class="neos-controls">
-				<f:form.textfield property="title" id="title" class="neos-span3" autofocus="true"/>
+		<fieldset>
+			<div class="neos-control-group">
+				<label class="neos-control-label" for="title">{neos:backend.translate(id: 'workspaces.workspace.title', source: 'Modules', package: 'TYPO3.Neos')}</label>
+				<div class="neos-controls">
+					<f:form.textfield property="title" id="title" class="neos-span3" autofocus="true"/>
+				</div>
 			</div>
-		</div>
 
-		<div class="neos-control-group">
-			<label class="neos-control-label" for="description">{neos:backend.translate(id: 'workspaces.workspace.description', source: 'Modules', package: 'TYPO3.Neos')}</label>
-			<div class="neos-controls">
-				<f:form.textfield property="description" id="description" class="neos-span10"/>
+			<div class="neos-control-group">
+				<label class="neos-control-label" for="description">{neos:backend.translate(id: 'workspaces.workspace.description', source: 'Modules', package: 'TYPO3.Neos')}</label>
+				<div class="neos-controls">
+					<f:form.textfield property="description" id="description" class="neos-span10"/>
+				</div>
 			</div>
-		</div>
 
-		<div class="neos-control-group{f:validation.ifHasErrors(for: 'baseWorkspace', then: ' neos-error')}">
-			<label class="neos-control-label" for="baseWorkspace">{neos:backend.translate(id: 'workspaces.workspace.baseWorkspace', source: 'Modules', package: 'TYPO3.Neos')}</label>
-			<div class="neos-controls">
-				<f:if condition="{disableBaseWorkspaceSelector}">
-					<f:then>
-						<f:form.select property="baseWorkspace" id="baseWorkspace" options="{baseWorkspaceOptions}" disabled="disabled" title="{neos:backend.translate(id: 'workspaces.help.cantRebaseWorkspaceContainsChanges', source: 'Modules', package: 'TYPO3.Neos')}" />
-					</f:then>
-					<f:else>
-						<f:form.select property="baseWorkspace" id="baseWorkspace" options="{baseWorkspaceOptions}" />
-					</f:else>
-				</f:if>
+			<div class="neos-control-group{f:validation.ifHasErrors(for: 'baseWorkspace', then: ' neos-error')}">
+				<label class="neos-control-label" for="baseWorkspace">{neos:backend.translate(id: 'workspaces.workspace.baseWorkspace', source: 'Modules', package: 'TYPO3.Neos')}</label>
+				<div class="neos-controls">
+					<f:if condition="{disableBaseWorkspaceSelector}">
+						<f:then>
```

Comment:
this diff is rather useless, there's nothing changed except for indenting two spaces due to the added fieldset wrap tag